Sources/Preferences/Localization.swift

```diff
@@ -99,26 +89,42 @@ struct Localization {
 	- Parameter identifier: Identifier of the string to localize.
 	*/
 	static func get(identifier: Identifier) -> String {
-		let locale = Locale.current
-
 		// Force-unwrapped since all of the involved code is under our control.
 		let localizedDict = Localization.localizedStrings[identifier]!
 		let defaultLocalizedString = localizedDict["en"]!
 
-		guard
-			let languageCode = locale.languageCode,
-			let regionCode = locale.regionCode
-		else {
-			return defaultLocalizedString
+		// Iterate through all user-preferred languages until we find one that has a valid language code
+		var preferredLocale: Locale?
```

Comment:
```suggestion
		var preferredLocale = Locale.current
```

This would simplify some code.

Reply:
Done with the latest commit.